tar_includes/README

```diff
@@ -1,17 +1,39 @@
 Thanks for using ungoogled-chromium! You have downloaded a Portable Linux build.
 
+SANDBOXING
 Before you run, you must setup your system so the browser's sandboxing will work. There are two options:
 * Use the user namespace sandbox. This is the preferred option.
 * Use the SUID sandbox. This should be used on systems that do not have support for user namespaces.
 
-How to setup the SUID sandbox:
+To quickly determine your current sandbox setup:
+
+1. Launch UC temporarily from a terminal:
```

Comment:
Might be a bit clearer if the instructions introduce UC to be an abbreviation of ungoogled-chromium, especially since ungoogled-chromium does not have an official abbreviation.

Reply:
Agreed.  Perhaps the most relevant place to introduce the abbreviation might be in the first sentence of this README webpage, as follows, so that the abbreviation could be used anywhere thereafter:-

_Portable Linux (i.e. a generic Linux version) packaging for ungoogled-chromium (UC)._

I won't amend that line after all, seeing how the proposed occurrences of UC abbreviations have now been removed and you point out anyway that there is no official abbreviation.  It looks like there could be a browser name change anyway.